catapult-sdk/src/plugins/lockSecret.js

```diff
@@ -46,10 +46,9 @@ const lockSecretPlugin = {
 		});
 
 		builder.addTransactionSupport(EntityType.secretLock, {
-			mosaicId: ModelType.uint64HexIdentifier,
-			amount: ModelType.uint64,
-			duration: ModelType.uint64,
 			secret: ModelType.binary,
+			mosaic: ModelType.uint64HexIdentifier,
```

Comment:
this schema has not changed

it is true that marti changed the SDK serializer and removed the `amount` due to a confusion and we have reverted this, but the linked comment here refers to a REST schema, which we have still not worked on (starting today) - i believe we cannot test this collection without the SDK because we have no secretlocks in our database, but still, this has not changed in the schemas, so most likely it is fine as it is